CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v2.15.0
+------------------------------
+*January 24, 2019*
+
+### Added
+- Cookie banner component page
+
```

Comment:
And **nit**: extra whitespace while you're changing the date please! 
_sorry_

Reply:
@xander-marjoram and this too :)